Gemfile

```diff
@@ -9,6 +9,7 @@ gem 'nokogiri'
 ### Fastlane Plugins
 
 gem 'fastlane-plugin-wpmreleasetoolkit', '~> 9.2'
+gem 'fastlane-plugin-sentry'
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPress-Android/blob/f132e700785ba69cbb9320f8793e693509b3a5ba/Gemfile#L12"></span>Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem <code>fastlane-plugin-sentry</code> should appear before <code>fastlane-plugin-wpmreleasetoolkit</code>.</td>
    </tr>
  </tbody>
</table>